corehq/apps/users/views/mobile/users.py

```diff
@@ -159,10 +159,10 @@ def page_context(self):
             'data_fields_form': self.custom_data.form,
             'can_use_inbound_sms': domain_has_privilege(self.domain, privileges.INBOUND_SMS),
         }
-        if self.request.project.commtrack_enabled or self.request.project.locations_enabled:
+        if self.request.project.commtrack_enabled or self.domain_object.uses_locations:
```

Comment:
They should point to the same thing.  `domain_object` is defined [here](https://github.com/dimagi/commcare-hq/blob/locations-for-robber-barons/corehq/apps/domain/views.py#L179-185), I think most reports have it available.